tests/API_RH.py

```diff
@@ -0,0 +1,49 @@
+# Import and load necessary lib
+from bs4 import BeautifulSoup
```

Comment:
This is a copy of API_FB. Is there a reason to commit it before modifying it to work for RH?

Reply:
Sorry I don't have permission to define this PR as a draft.. It fixed and will be uploaded in next hour